templates/shared_services/airlock_notifier/porter.yaml

```
@@ -44,13 +44,21 @@ parameters:
   - name: smtp_server_address
     env: SMTP_SERVER_ADDRESS
     type: string
+  - name: smtp_server_port
+    env: SMTP_SERVER_PORT
+    type: string
+    default: "25"
   - name: smtp_username
     env: SMTP_USERNAME
     type: string
   - name: smtpPassword
     env: SMTP_PASSWORD
     type: string
     sensitive: true
+  - name: smtp_server_enable_ssl
+    env: SMTP_SERVER_ENABLE_SSL
+    type: boolean
+    default: false
```

I think this should be 
```suggestion
    default: true
```

---

Perfect. I have just accepted the suggestion.